PCbuild/build.bat

```diff
@@ -115,7 +115,8 @@ if exist "%GIT%" set GITProperty=/p:GIT="%GIT%"
 if not exist "%GIT%" echo Cannot find Git on PATH & set GITProperty=
 
 rem Setup the environment
-call "%dir%env.bat" %vs_platf% >nul
```

Comment:
But then I think the cross-compilation for PGO warning has to go away too... not sure of the full impact there yet, let me try it.